lib/post/widgets/post_view.dart

```diff
@@ -228,6 +255,49 @@ class _PostSubviewState extends State<PostSubview> with SingleTickerProviderStat
                     hasBeenEdited: postViewMedia.postView.post.updated != null ? true : false,
                     url: postViewMedia.media.firstOrNull != null ? postViewMedia.media.first.originalUrl : null,
                   ),
+                  if (widget.showReplyEditorButtons && widget.postViewMedia.postView.post.body?.isNotEmpty == true) ...[
```

Comment:
It seems like this logic and the comment reply action logic is the same - we can move this to its own widget

Reply:
Sure! I left the divider there, since it feels separate to me, but I moved the row with the buttons.